.circleci/config.yml

```diff
@@ -17,13 +25,26 @@ jobs:
               pip install pip tox --upgrade
       - run:
           name: Run tests
-          command: tox -e py39-test-image-mpl311 -- -P visualization --remote-data=astropy --open-files --mpl --mpl-results-path=$PWD/results -W ignore:np.asscalar -W ignore::DeprecationWarning -k "not test_no_numpy_warnings"
+          command: tox -v -- -W ignore:np.asscalar -W ignore::DeprecationWarning -k "not test_no_numpy_warnings"
```

Comment:
Why are we only ignoring these warnings on circle and not in `tox.ini`?

Reply:
Fixed